ui/src/app/applications/components/application-details/application-details.tsx

```diff
@@ -158,209 +161,227 @@ export class ApplicationDetails extends React.Component<RouteComponentProps<{nam
                                         part + ' '
                                     )
                                 );
+                            const clustersPromise = services.clusters.list().then(clusters => clusters.sort());
```

Comment:
Do we need to sort the clusters?

Reply:
I see other places we get clusters it's always sorted, so I kept it here too. Is there any concerns about this sorting?

Reply:
Removed.